Allow platform-specific exclusions to be selected independently.
Upstream's build state for these programs is otherwise tied directly to the
global default, a broader utility group, or an automatic build check.

--- a/configure.ac
+++ b/configure.ac
@@ -1907,7 +1907,11 @@
 AM_CONDITIONAL([BUILD_LSLOGINS], [test "x$build_lslogins" = xyes])
 
 
-UL_BUILD_INIT([chcpu], [check])
+AC_ARG_ENABLE([chcpu],
+  AS_HELP_STRING([--disable-chcpu], [do not build chcpu]),
+  [], [UL_DEFAULT_ENABLE([chcpu], [check])]
+)
+UL_BUILD_INIT([chcpu])
 UL_REQUIRES_LINUX([chcpu])
 UL_REQUIRES_HAVE([chcpu], [cpu_set_t], [cpu_set_t type])
 AM_CONDITIONAL([BUILD_CHCPU], [test "x$build_chcpu" = xyes])
@@ -1979,7 +1983,11 @@
 UL_REQUIRES_LINUX([pipesz])
 AM_CONDITIONAL([BUILD_PIPESZ], [test "x$build_pipesz" = xyes])
 
-UL_BUILD_INIT([fadvise], [check])
+AC_ARG_ENABLE([fadvise],
+  AS_HELP_STRING([--disable-fadvise], [do not build fadvise]),
+  [], [UL_DEFAULT_ENABLE([fadvise], [check])]
+)
+UL_BUILD_INIT([fadvise])
 UL_REQUIRES_LINUX([fadvise])
 AM_CONDITIONAL([BUILD_FADVISE], [test "x$build_fadvise" = xyes])
 
@@ -2161,7 +2169,11 @@
 UL_REQUIRES_LINUX([choom])
 AM_CONDITIONAL([BUILD_CHOOM], [test "x$build_choom" = xyes])
 
-UL_BUILD_INIT([lsipc], [check])
+AC_ARG_ENABLE([lsipc],
+  AS_HELP_STRING([--disable-lsipc], [do not build lsipc]),
+  [], [UL_DEFAULT_ENABLE([lsipc], [check])]
+)
+UL_BUILD_INIT([lsipc])
 UL_REQUIRES_LINUX([lsipc])
 UL_REQUIRES_BUILD([lsipc], [libsmartcols])
 AM_CONDITIONAL([BUILD_LSIPC], [test "x$build_lsipc" = xyes])
@@ -2196,7 +2208,11 @@
 UL_BUILD_INIT([setsid], [yes])
 AM_CONDITIONAL([BUILD_SETSID], [test "x$build_setsid" = xyes])
 
-UL_BUILD_INIT([readprofile], [check])
+AC_ARG_ENABLE([readprofile],
+  AS_HELP_STRING([--disable-readprofile], [do not build readprofile]),
+  [], [UL_DEFAULT_ENABLE([readprofile], [check])]
+)
+UL_BUILD_INIT([readprofile])
 UL_REQUIRES_LINUX([readprofile])
 AM_CONDITIONAL([BUILD_READPROFILE], [test "x$build_readprofile" = xyes])
 
@@ -2226,7 +2242,11 @@
 UL_REQUIRES_SYSCALL_CHECK([exch], [UL_CHECK_SYSCALL([renameat2])])
 AM_CONDITIONAL([BUILD_EXCH], [test "x$build_exch" = xyes])
 
-UL_BUILD_INIT([fincore], [check])
+AC_ARG_ENABLE([fincore],
+  AS_HELP_STRING([--disable-fincore], [do not build fincore]),
+  [], [UL_DEFAULT_ENABLE([fincore], [check])]
+)
+UL_BUILD_INIT([fincore])
 UL_REQUIRES_LINUX([fincore])
 UL_REQUIRES_BUILD([fincore], [libsmartcols])
 AM_CONDITIONAL([BUILD_FINCORE], [test "x$build_fincore" = xyes])
@@ -2249,11 +2269,19 @@
 UL_REQUIRES_HAVE([blkpr], [linux_pr_h], [linux/pr.h header])
 AM_CONDITIONAL([BUILD_BLKPR], [test "x$build_blkpr" = xyes])
 
-UL_BUILD_INIT([ldattach], [check])
+AC_ARG_ENABLE([ldattach],
+  AS_HELP_STRING([--disable-ldattach], [do not build ldattach]),
+  [], [UL_DEFAULT_ENABLE([ldattach], [check])]
+)
+UL_BUILD_INIT([ldattach])
 UL_REQUIRES_LINUX([ldattach])
 AM_CONDITIONAL([BUILD_LDATTACH], [test "x$build_ldattach" = xyes])
 
-UL_BUILD_INIT([rtcwake], [check])
+AC_ARG_ENABLE([rtcwake],
+  AS_HELP_STRING([--disable-rtcwake], [do not build rtcwake]),
+  [], [UL_DEFAULT_ENABLE([rtcwake], [check])]
+)
+UL_BUILD_INIT([rtcwake])
 UL_REQUIRES_LINUX([rtcwake])
 AM_CONDITIONAL([BUILD_RTCWAKE], [test "x$build_rtcwake" = xyes])
 
@@ -2269,17 +2297,26 @@
   [], [UL_DEFAULT_ENABLE([scriptutils], [check])]
 )
 
-UL_ENABLE_ALIAS([script], [scriptutils])
+AC_ARG_ENABLE([script],
+  AS_HELP_STRING([--disable-script], [do not build script]),
+  [], [UL_DEFAULT_ENABLE([script], [check])]
+)
 UL_BUILD_INIT([script])
 UL_REQUIRES_HAVE([script], [pty], [openpty function (libutil)])
 AM_CONDITIONAL([BUILD_SCRIPT], [test "x$build_script" = xyes])
 
-UL_ENABLE_ALIAS([scriptreplay], [scriptutils])
+AC_ARG_ENABLE([scriptreplay],
+  AS_HELP_STRING([--disable-scriptreplay], [do not build scriptreplay]),
+  [], [UL_DEFAULT_ENABLE([scriptreplay], [check])]
+)
 UL_BUILD_INIT([scriptreplay])
 AM_CONDITIONAL([BUILD_SCRIPTREPLAY], [test "x$build_scriptreplay" = xyes])
 
-UL_ENABLE_ALIAS([scriptlive], [scriptutils])
-UL_BUILD_INIT([scriptlive])
+AC_ARG_ENABLE([scriptlive],
+  AS_HELP_STRING([--disable-scriptlive], [do not build scriptlive]),
+  [], [enable_scriptlive=$enable_scriptutils]
+)
+UL_BUILD_INIT([scriptlive], [], [scriptlive])
 UL_REQUIRES_HAVE([scriptlive], [pty], [openpty function (libutil)])
 AM_CONDITIONAL([BUILD_SCRIPTLIVE], [test "x$build_scriptlive" = xyes])
 
@@ -2629,7 +2666,10 @@
 AM_CONDITIONAL([BUILD_TASKSET], [test "x$build_taskset" = xyes])
 
 
-UL_ENABLE_ALIAS([coresched], [schedutils])
+AC_ARG_ENABLE([coresched],
+  AS_HELP_STRING([--disable-coresched], [do not build coresched]),
+  [], [enable_coresched=$enable_schedutils]
+)
 UL_BUILD_INIT([coresched])
 UL_REQUIRES_SYSCALL_CHECK([coresched],
 	[UL_CHECK_SYSCALL([prctl])],
@@ -2650,7 +2690,10 @@
 	UL_CHECK_SYSCALL([sched_setattr])
 ])
 
-UL_ENABLE_ALIAS([uclampset], [schedutils])
+AC_ARG_ENABLE([uclampset],
+  AS_HELP_STRING([--disable-uclampset], [do not build uclampset]),
+  [], [enable_uclampset=$enable_schedutils]
+)
 UL_BUILD_INIT([uclampset])
 UL_REQUIRES_SYSCALL_CHECK([uclampset],
 	[UL_CHECK_SYSCALL([sched_setattr])], [sched_setattr])
